Sources/WebDriver/Requests.swift

```diff
@@ -524,6 +524,41 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidlocation
+    public enum SessionGeoLocation {
+        public struct Post: Request {
+            public var session: String
+            public var latitude: Int
+            public var longitude: Int
+            public var altitude: Int
+
+            public var pathComponents: [String] {["session", session, "location"]}
```

Comment:
Please include spaces inside single-line curly braces throughout.
```suggestion
            public var pathComponents: [String] { ["session", session, "location"] }
```